tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -855,3 +855,6 @@ user removes key stat
     [Arguments]    ${tile_num}
     user waits until page contains element    xpath://*[@data-testid="keyStat"][${tile_num}]
     user clicks element    xpath://*[@data-testid="keyStat"][${tile_num}]//button[contains(text(), "Remove")]
+
+user closes user testing modal
+    user clicks element    //*[@data-testid="admin-survey-banner"]//button[text()="Close"]
```

Comment:
You could change this to `user closes user testing modal if exists` and use `user clicks element if exists` - then tests won't break when this modal gets removed.